components/automate-ui/src/app/modules/user/user-details/user-details.component.html

```diff
@@ -17,62 +9,38 @@
       <chef-page-header>
         <div class="detail-row">
           <div class="name-column">
-            <ng-container *ngIf="!updatingUser">
-              <div class="display3">{{ user?.name }}</div>
-            </ng-container>
-            <ng-container *ngIf="updatingUser">
-              <form [formGroup]="editForm">
-                <chef-form-field class="display3">
-                  <label>
-                    <span class="label">Full Name <span aria-hidden="true">*</span></span>
-                    <input chefInput formControlName="fullName"/>
-                  </label>
-                  <chef-error
-                    *ngIf="(editForm.get('fullName').hasError('required') || editForm.get('fullName').hasError('pattern')) && editForm.get('fullName').dirty">
-                    Full Name is required.
-                  </chef-error>
-                </chef-form-field>
-              </form>
-            </ng-container>
-            <div class="container-big-header inline-container">
-              <span class="text"> {{ user?.id }}</span>
-            </div>
-          </div>
-          <div class="button-column">
-            <ng-container *ngIf="!updatingUser">
-              <app-authorized [allOf]="[['/auth/users/{username}', 'put', user?.id], ['/iam/v2/users/{username}', 'put', user?.id],
-                ['/auth/users/{username}', 'get', user?.id], ['/iam/v2/users/{username}', 'get', user?.id]]">
-                <chef-button secondary class="edit-button" (click)="setUpdatingUser(true)">
-                  <chef-icon>mode_edit</chef-icon>
-                  <span>Edit</span>
-                </chef-button>
-              </app-authorized>
-              <app-authorized [allOf]="[['/auth/users/{username}', 'delete', user?.id], ['/iam/v2/users/{username}', 'delete', user?.id],
-                ['/auth/users/{username}', 'get', user?.id], ['/iam/v2/users/{username}', 'get', user?.id]]">
-                <chef-button *ngIf="isAdminView" secondary caution class="delete-button"
-                  (click)="openDeleteConfirmationModal()">
-                  <chef-icon>delete</chef-icon>
-                  <span>Delete user</span>
-                </chef-button>
-              </app-authorized>
-            </ng-container>
-            <ng-container *ngIf="updatingUser">
-              <chef-button primary class="save-button" [disabled]="!editForm.valid || !editForm.dirty"
-                (click)="saveFullName()">
-                <span>Save</span>
-              </chef-button>
-              <chef-button tertiary (click)="setUpdatingUser(false)">Cancel</chef-button>
-            </ng-container>
+            <div class="header-user-name" *ngIf="!isAdminView">Profile: {{ user?.name }}</div>
+            <div class="header-user-name" *ngIf="isAdminView">{{ user?.name }}</div>
+            <div class="header-user-id-label">Username</div>
```

Comment:
- there is `25px` of space between the top content and the tabs, this should be `15px`
- also, you are using divs here, but the other pages use a table... not sure if anyone cares...

Reply:
I am changing the spacing to 15px. Could you should me how to use zeplin sometime. Below is an image where I tried to find the spacing and it is showing 30px :man_shrugging:. 
![image](https://user-images.githubusercontent.com/1679247/72671391-4debd900-39fe-11ea-917c-52aebcde357b.png)

Reply:
Those designs are wrong or they weren't implemented to match previously, that's my fault. I was just looking at the other pages to compare. Let's sync Tuesday though.